style.css

```
@@ -288,3 +288,11 @@ section.coc {
     padding: 2em 2em 2em 55px;
   }
 }
+
+/*
+ * elm-markdownのMarkdownの中にHTMLタグがあると、
+ * その直下に<p>タグが必ず入るよう(<b>abc</b> → <b><p>abc</p></b>となる)なのでCSS上は何もない扱いにする
+ */
+.markdown-html-workaround > * p {
+  display: inline;
```

瑣末ですが、コメントの意図により近い値として `contents` があるので、これを使うと良さそうです。
余分なホワイトスペースも省略できる。

```suggestion
  display: contents;
```

---

なるほど、良いですね！🍹